zitadel/terraform/environments/environment.tfvars

```diff
@@ -1,2 +1,3 @@
 xpanse-ui_base_uri = "http://localhost:3000"
```

Comment:
@jinyangyang222 i just realized that we do not need this extra environment.tfvars file. We need only local.tfvars and testbed.tfvars. 
the current values in  environment.tfvars can be moved to local.tfvars
and to the testbed.tfvars, we can add the following "https://demo.xpanse.site" value for both URIs for now. 

Please also remove the references to "environment.tfvars" file in readme.

Reply:
updated.